### PR TITLE
Awesome books: a complete website with navigation PR

### PR DESCRIPTION
**Pull request by belmeetmule**
### In this PR, I and my coding partner have done the following tasks:

- [ ] Refactor our js 
- [ ] add contact and add new pages
- [ ] add navigation and footer to all the pages
- [ ] Worked on local storage
- [ ] Worked on DOM manipulation and js classes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,17 +10,28 @@
 </head>
 <body>
     <header>
-        <h1>All awesome books</h1>
+        <nav class="nav-bar">
+            <p class="logo">Awesome Books</p>
+            <ul class="menu">
+                <li class="menu-item" id="list" >List</li>
```
Comment (topeogunleye) [inline index.html:16]: - [ ] As written in the project requirements, Kindly ensure that when any of the links are clicked, the main area changes to display only one of the three content sections. you can add and remove CSS classes to display or hide the content by using the [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) add and remove methods.


[Video Demo](https://www.loom.com/share/3e7a93ab5b264fcf8f2c5366723b30bd)

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @belmeetmule, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your @topeogunleye so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css [hunk 2]
```diff
@@ -60,3 +116,29 @@ h1 {
   background-color: #d3d3d3;
   color: #000;
 }
+
+.foot {
+  position: fixed;
+  bottom: 0;
+  width: 100%;
+  background-color: #fff;
```
Comment (Ugboaja-Uchechi) [inline index.css:124]: - [ ] [OPTIONAL] Kindly make sure you space the list of books for a better user experience. 🙏 

image

**Review by Ugboaja-Uchechi — APPROVED** [review]
Hi @belmeetmule and @OmarMWarraich , 

## Status: APPROVED 🥳 🥳 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 
- Great job with your design and functionalities 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**